db/migrate/20200721230149_create_games.rb

```diff
@@ -0,0 +1,32 @@
+class CreateGames < ActiveRecord::Migration[5.2]
+  def change
+    create_table :games do |t|
+      t.string :title
+      t.integer :age_rating
+      t.float :agg_rating
+      t.text :artworks, array: true, default: []
```

Comment:
A lot of the values returned from the API are in array format. This syntax should allow us to shovel values in

Reply:
Shoveling in values is a pretty clever move. 💯